### PR TITLE
Preload `GitRepositoryComboBox` class so that it does not end up loaded from action update()

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseProjectDependentAction.java [hunk 1]
```diff
@@ -22,12 +26,17 @@
...
-    @SuppressWarnings("nullness:argument") val dummyService = new SideEffectingActionTrackingService(null);
+    preloadClasses();
+  }
 
-    @SuppressWarnings("nullness:argument") val dummyId = new SideEffectingActionTrackingService.SideEffectiveActionId(null);
+  @SneakyThrows
+  private static void preloadClasses() {
+    for (String className : IOUtils.resourceToString("/classes-to-preload.txt", StandardCharsets.UTF_8).lines().toList()) {
+      Class.forName(className);
+    }
```
[the remark this answers is not in the document]
Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseProjectDependentAction.java:39]: It's okay, it's called solely from `static` block, any UI test will capture that instantly

File: frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/gitrepositoryselection/GitRepositoryComboBox.java [hunk 2]
```diff
@@ -46,7 +47,7 @@ public GitRepositoryComboBox(Project project) {
 
     val messageBusConnection = project.getMessageBus().connect();
     messageBusConnection
-        .subscribe(VcsRepositoryManager.VCS_REPOSITORY_MAPPING_UPDATED, () -> {
+        .<VcsRepositoryMappingListener>subscribe(VcsRepositoryManager.VCS_REPOSITORY_MAPPING_UPDATED, () -> {
           LOG.debug("Git repository mappings changed");
           ModalityUiUtil.invokeLaterIfNeeded(ModalityState.NON_MODAL, () -> updateRepositories());
         });
```
[the remark this answers is not in the document]
Comment (PawelLipski) [reply frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/gitrepositoryselection/GitRepositoryComboBox.java:53]: 🤔 I suppose this is handled by IntelliJ behind the scenes somehow

File: frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/gitrepositoryselection/GitRepositoryComboBox.java [hunk 3]
```diff
@@ -46,7 +47,7 @@ public GitRepositoryComboBox(Project project) {
 
     val messageBusConnection = project.getMessageBus().connect();
     messageBusConnection
-        .subscribe(VcsRepositoryManager.VCS_REPOSITORY_MAPPING_UPDATED, () -> {
+        .<VcsRepositoryMappingListener>subscribe(VcsRepositoryManager.VCS_REPOSITORY_MAPPING_UPDATED, () -> {
```
Comment (PawelLipski) [inline frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/gitrepositoryselection/GitRepositoryComboBox.java:50]: Unrel, IntelliJ was displaying red squiggles here

**Review by mkondratek — APPROVED** [review]

**Comment by mkondratek** [comment]
niezłe haksy

**Pull request closed without merging**